compiler/test/compilable/transition_in.d

```diff
@@ -5,7 +5,7 @@ TEST_OUTPUT:
 ---
 compilable/transition_in.d(3): Usage of 'in' on parameter
 compilable/transition_in.d(3): Usage of 'in' on parameter
-compilable/transition_in.d(8): Usage of 'in' on parameter
+compilable/transition_in.d(13): Usage of 'in' on parameter
 compilable/transition_in.d(13): Usage of 'in' on parameter
```

Comment:
Somethings going wrong here for you to end up with a double error.

Reply:
It's probably running semantic more than once on a tree, which happens in the compiler. Not worth dealing with it in this PR.

Reply:
After all, this PR does not cause semantic to be run twice, that was already happening.